features/step_definitions/appointment_locator_steps.rb

```diff
@@ -37,3 +41,21 @@
     expect(location.public_send(element)).to be_visible
   end
 end
+
+Then(/^I should see the following appointment location details:$/) do |table|
+  location = Pages::Location.new
+
+  table.rows.flatten.each do |detail|
+    attribute = case detail
+                when 'its name'
+                  :name
+                when 'its address'
+                  :address
+                when 'its opening hours'
+                  :hours
+                when 'its Pension Wise booking phone number'
+                  :phone
+                end
+    expect(location.public_send(attribute)).to be_visible
```

Comment:
If we can assert the actual values then we get a slightly more complete edge-to-edge test. Perhaps worth it?